src/main/java/redis/clients/jedis/resps/GeoRadiusResponse.java

```diff
@@ -44,4 +46,20 @@ public long getRawScore() {
   public void setRawScore(long rawScore) {
     this.rawScore = rawScore;
   }
+
+  @Override
+  public boolean equals(Object obj) {
```

Comment:
This is a new feature. Add proper unit tests for it.

Reply:
Added [here](https://github.com/AvitalFineRedis/jedis/pull/1)